src/name_bazaar/ui/db.cljs

```diff
@@ -11,21 +11,28 @@
     [re-frame.core :refer [dispatch]]
     [district0x.ui.utils :as d0x-ui-utils]))
 
+(goog-define environment "production")
+
 (def development-config
   {:node-url "http://localhost:8549"
    :load-node-addresses? true
+   :root-url "https://beta.namebazaar.io"
```

Comment:
Why not to detect this programmatically like from window.location object?

Reply:
Just to make sure we have a canonical name in posts - probably window.location will be the same, but i didn't want to couple them this way.

Reply:
okay